cmd/temporal/main.go

```diff
@@ -584,6 +585,10 @@ func main() {
 		Version = "latest"
 	}
 
+	// allow 1024 bit key on ipfs
```

Comment:
this needs to be removed as it's insecure. 1024 bit rsa keys are no longer considered secure, and 2048 bit is the bare-minimum we should accept.

Reply:
Change removed. I did this because CI was reporting key length errors from the bootstrap list, but this change did not fix the issue.